manifests/monitoring/pushgateway.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: pushgateway
+  namespace: monitoring
+  annotations:
+    prometheus.io/scrape: 'true'
+  labels:
+    k8s-app: pushgateway
+spec:
+  selector:
+    k8s-app: pushgateway
+  type: NodePort
+  ports:
+    - name: pushgateway
+      protocol: TCP
+      port: 9091
+
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: pushgateway
+  namespace: monitoring
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      k8s-app: pushgateway
+  template:
+    metadata:
+      name: pushgateway
+      labels:
+        k8s-app: pushgateway
+    spec:
+      containers:
+        - name: pushgateway
+          image: prom/pushgateway:latest
+          ports:
+            - name: web
+              containerPort: 9091
+---
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  labels:
+    k8s-app: pushgateway
+  name: pushgateway
+  namespace: monitoring
+spec:
+  rules:
+    - host: pushgateway.{{.name}}.lab.flanksource.com
```

Comment:
This should be `pushgateway.{{.domain}}`

Reply:
> Can we also a short description to the command on how the test works ?

Sure